### DISCUSSION TITLE
How can I integrate my own custom 3D model built on threejs with various textures like my own, draco, ktx2, and dracoktx2?

### DISCUSSION THREAD

**Discussion by Jayy0906** [google/model-viewer#4346] [Q&A]
How can I perform an automated test to compare my single threejs model's ss and obtain a result in dB value without needing to use any additional web renderers, such as babylon or gltf, as I indicated in the title?

**Comment by elalish** [comment]
I'm not exactly sure what you're asking, but I'd recommend looking at our `render-fidelity-tools` package to see how we automate tests with dB metrics.
